app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  include Pagy::Frontend
+  
+  def sortable(column, title = nil, temp)
```

Comment:
Update  sortable(column, title = nil, temp) to  sortable(column,  temp, title = nil)